### PR TITLE
Setup project and automation

### PR DESCRIPTION
**Pull request by dkondyrev**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get upgrade -y
```
Comment (rvadim) [inline Dockerfile]: There are no need for upgrade I think.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:18.04
```
Comment (rvadim) [inline Dockerfile]: Just `FROM python:3.7` pip already included.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get -y install python3.6 python3-pip
+
+WORKDIR /home/kubernetes
+
```
Comment (rvadim) [inline Dockerfile:8]: Need to copy main.py to container

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:18.04
+
```
Comment (rvadim) [inline Dockerfile:2]: Need labels: maintainer, source, description

File: deploy.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
```
Comment (rvadim) [inline deploy.yaml]: Let's put this file to separate directory  `deploy` or something.

**Review by rvadim — CHANGES_REQUESTED** [review]
Good start! Let's fix some minor issues with Dockerfile

File: Dockerfile (hunk 1 above)
Comment (dkondyrev) [reply Dockerfile]: Update and upgrade commands are removed

File: Dockerfile (hunk 2 above)
Comment (dkondyrev) [reply Dockerfile]: Changed
```suggestion
FROM python:3.7
```

File: Dockerfile (hunk 3 above)
Comment (dkondyrev) [reply Dockerfile:8]: Fixed

File: Dockerfile (hunk 4 above)
Comment (dkondyrev) [reply Dockerfile:2]: Fixed

File: deploy.yaml (hunk 5 above)
Comment (dkondyrev) [reply deploy.yaml]: Fixed

**Review by rvadim — APPROVED** [review]
LGTM

**Review by Kuyantsev-VP — APPROVED** [review]

**Comment by rvadim** [comment]
> I think I should create my pull request after this pull request will be merged. Am I right?
yep

**Pull request merged**